app/models/concerns/edition/scopes.rb

```diff
@@ -0,0 +1,158 @@
+module Edition::Scopes
+  extend ActiveSupport::Concern
+
+  included do
+    scope :with_title_or_summary_containing,
+          lambda { |*keywords|
+            pattern = "(#{keywords.map { |k| Regexp.escape(k) }.join('|')})"
+            in_default_locale.where("edition_translations.title REGEXP :pattern OR edition_translations.summary REGEXP :pattern", pattern:)
+          }
+
+    scope :with_title_containing,
+          lambda { |keywords|
+            escaped_like_expression = keywords.gsub(/([%_])/, "%" => '\\%', "_" => '\\_')
+            like_clause = "%#{escaped_like_expression}%"
+
+            scope = in_default_locale.includes(:document)
+            scope
+              .where("edition_translations.title LIKE :like_clause", like_clause:)
+              .or(scope.where(document: { slug: keywords }))
+          }
+
+    scope :in_pre_publication_state, -> { where(state: Edition::PRE_PUBLICATION_STATES) }
+    scope :force_published, -> { where(state: "published", force_published: true) }
+    scope :not_published, -> { where(state: %w[draft submitted rejected]) }
+    scope :without_not_published, -> { where.not(state: %w[draft submitted rejected]) }
```

Comment:
In general I'd be keen to see us refactor into domain based concerns rather than technical concerns. So for these scopes, I'd suggest factoring them into the edition workflow concern. I realise this may not be easy everywhere but where the opportunity arises I feel we should take it.